pkg/services/serviceaccounts/database/database.go

```diff
@@ -18,6 +20,23 @@ func NewServiceAccountsStore(store *sqlstore.SQLStore) *ServiceAccountsStoreImpl
 	}
 }
 
+func (s *ServiceAccountsStoreImpl) CreateServiceAccount(ctx context.Context, sa *serviceaccounts.CreateServiceaccountForm) (user *models.User, err error) {
+	// create a new service account - "user" with empty permissions
+	cmd := models.CreateUserCommand{
+		Login:            "Service-Account-" + uuid.New().String(),
+		Email:            uuid.New().String(),
+		Password:         "Password-" + uuid.New().String(),
```

Comment:
```suggestion
```
nit: should probably remove the need for a email and password for a serviceaccount